src/scenes/NurseriesRouter/index.tsx

```diff
@@ -1,8 +1,7 @@
 import React, { useCallback } from 'react';
-import { Route, Switch } from 'react-router-dom';
+import { Route, Routes } from 'react-router-dom';
 
 import EmptyStatePage from 'src/components/emptyStatePages/EmptyStatePage';
-import { APP_PATHS } from 'src/constants';
```

Comment:
Why get rid of APP_PATHS and the hardcoded route paths below?

Reply:
Ah, because these paths are just fragments of the original url.. makes sense. Kinda sad we can't use the same constants now, but it's a react change.

Reply:
Yes, now if a the routers is a subrouter, the first part of the url is already "included"